scripts/listapranzo.js

```diff
@@ -85,6 +85,26 @@ module.exports = function (robot) {
     order.users[user.id] = dishes;
   };
 
+  var fuzzyMatch = function (dish, menuline) {
+    var key = dish.trim().toLowerCase().replace(" ", ".*") + ".*";
+    
+    key = new RegExp(key, "");
```

Comment:
beh potrebbe wrappare il tutto direttamente in regexp ma poi diventa lungo come la morte.. e fare due variabili non serve a niente. io approvo :P